src/App.tsx

```diff
@@ -1,52 +1,95 @@
+import {
+  Navigate,
+  NavLink,
+  Route,
+  Routes,
+} from 'react-router-dom';
+
+import cn from 'classnames';
+import { TabsPage } from './TabsPage';
+
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
 
-// const tabs = [
-//   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
-//   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
-//   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
-// ];
-
-export const App = () => (
-  <>
-    {/* Also requires <html class="has-navbar-fixed-top"> */}
-    <nav
-      className="navbar is-light is-fixed-top is-mobile has-shadow"
-      data-cy="Nav"
-    >
-      <div className="container">
-        <div className="navbar-brand">
-          <a href="/" className="navbar-item is-active">Home</a>
-          <a href="/tabs" className="navbar-item">Tabs</a>
-        </div>
-      </div>
-    </nav>
-
-    <div className="section">
-      <div className="container">
-        <h1 className="title">Home page</h1>
-        <h1 className="title">Tabs page</h1>
-        <h1 className="title">Page not found</h1>
-
-        <div className="tabs is-boxed">
-          <ul>
-            <li data-cy="Tab" className="is-active">
-              <a href="#/">Tab 1</a>
-            </li>
-            <li data-cy="Tab">
-              <a href="#/">Tab 2</a>
-            </li>
-            <li data-cy="Tab">
-              <a href="#/">Tab 3</a>
-            </li>
-          </ul>
+enum Nav {
+  Home = 'Home',
+  Tabs = 'Tabs',
+}
+
+export const App = () => {
+  const renderNavLink = (text: Nav) => {
+    const path = {
+      Tabs: '/tabs',
+      Home: '/',
+    }[text];
+
+    return (
+      <NavLink
+        to={path}
+        className={({ isActive }) => (
+          cn('navbar-item', {
+            'is-active': isActive,
+          })
+        )}
+      >
+        {text}
+      </NavLink>
```

Comment:
In READ.me file it explicitly says not to use `NavLink`. Because only different between `NavLink` and `Link` is `Navlink` has an active state.

Reply:
I thought that was meant for TabsPage

Reply:
I says not to use it because is-active is on the parent element, which is true for the page links, but not the nav links

Reply:
👍🏻